app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
```

Comment:
Hmm, I'd prefer to split up the event types now. It'll only become more difficult to do it later, once we have a ton of data to migrate. What are your thoughts?